### ISSUE TITLE
Pull files list through pardot API

### ISSUE THREAD

**Issue by Prabakaran0312** [pardot/api-docs#171]
Hi

I would like to pull files(Images, pdf etc) list using Pardot API. But i cant find the API URL information below "Version 3" and "Version 4" in this guide

http://developer.pardot.com/kb/api-version-3/accounts/

So could i know Is it possible to pull files list through API?

Please advice.

Thanks
Prabakaran

**Comment by creuzerm** [comment]
No, it is not possible at this time that I have been able to find.
